### DISCUSSION TITLE
Unable to run the cqube locally

### DISCUSSION THREAD

**Discussion by agamdahiya** [Sunbird-cQube/community#800] [Q&A]
I am facing the error when i was trying to setup the cQube locally . Facing the error in the Nifi installation.
Find the screenshot below.
![image](https://github.com/Sunbird-cQube/community/assets/127106035/65cac6b5-9dff-46cf-aa2a-2d6895c6a967)

**Comment by RNarmatha** [comment]
@padmanabha1234 - Please help on this

**Comment by RNarmatha** [comment]
@asifbasha61 - Please look into this

**Comment by asifbasha61** [comment]
@agamdahiya try building the nifi image manually and share the screenshot.
cd cqube-devops/microservices/nifi-ms
sudo docker build -t nifi_test .

**Comment by RNarmatha** [comment]
@agamdahiya - Kindly confirm was this issue fixed.
